docs/resources/deployment.md

```diff
@@ -123,7 +127,7 @@ resource "astro_deployment" "hybrid" {
 ### Read-Only
 
 - `airflow_version` (String) Deployment Airflow version
-- `astro_runtime_version` (String) Deployment Astro Runtime version. The terraform provider will use the latest Astro runtime version for the Deployment. The Astro runtime version can be updated with your Astro project Dockerfile
+- `astro_runtime_version` (String) Deployment's actual Astro Runtime version
```

Comment:
```suggestion
- `current_astro_runtime_version` (String) Deployment's current Astro Runtime version
```
I think `actual` makes it sound like the original one is unauthentic / fake, what do you think about `current`?

Reply:
this is a pretty good idea
will update

Reply:
I didn't add the `current_` prefix to the astro_runtime_verison because I think I feel like keeping it the same as the API would be better but definitely changed the description
great suggestion